src/index.html

```diff
@@ -7,6 +7,261 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="header__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <a class="nav__item is-active">Apple</a>
+          <a class="nav__item">Samsung</a>
+          <a class="nav__item">Smartphones</a>
+          <a class="nav__item" data-qa="nav-hover">laptops & computers</a>
+          <a class="nav__item">gadgets</a>
+          <a class="nav__item">tablets</a>
+          <a class="nav__item">photo</a>
+          <a class="nav__item">video</a>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="cards">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="card img"
+            class="card__image"
+          >
```

Comment:
```suggestion
          <img
            src="./images/imac.jpeg"
            alt="APPLE A1419 iMac 27"
            class="card__image"
          >
```

the alt attribute should describe what is in the image